packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -585,18 +585,21 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     return (
       <div
+        role={ 'button' }
         aria-labelledby={ item.ariaLabel }
+        className={ classNames.splitContainer }
         aria-disabled={ this._isItemDisabled(item) }
         aria-haspopup={ true }
         aria-describedby={ item.ariaDescription }
         aria-checked={ item.isChecked || item.checked }
         aria-posinset={ focusableElementIndex + 1 }
         aria-setsize={ totalItemCount }
+        onKeyDown={ this._onSplitContainerItemKeyDown.bind(this, item) }
+        onClick={ this._executeItemClick.bind(this, item) }
+        tabIndex={ 0 }
+        data-is-focusable={ true }
       >
-        <span
-          aria-hidden={ true }
-          className={ classNames.splitContainer }
-        >
+        <span aria-hidden={ true }>
```

Comment:
since the flex classname is on the div rather than the span, that just means that the span will flex to fit, but there's only one element so it will always be able to fit. Should this span have been deleted? The contents of the span should likely still be flexed.

Reply:
I put it back, because I wanted to have a way to have aria-hidden for all the child elements, but seeing as each of the pieces in the child element is uniquely created here, I don't see the harm in adding aria-hidden to each of the individual pieces.

Reply:
Is there any reason why we aren't just putting the span back to the way it was before this change (the more scoped we make the change the better, and making this change doesn't seem necessary)?